pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -82,11 +83,15 @@ func (hm *hostportManager) Add(id string, podPortMapping *PodPortMapping, natInt
 		return nil
 	}
 
-	if podPortMapping.IP.To4() == nil {
+	if podPortMapping.IP.To16() == nil {
```

Comment:
comment on the fact that this is safe for valid v4 IPs

Reply:
done